frontend/src/Components/ContentComponent.tsx

```diff
@@ -128,6 +134,31 @@ function processVideoEmbed(img: HTMLImageElement) {
     return !!videoUrl;
 }
 
+function updateVimeo(iframe: HTMLIFrameElement) {
+
+    loadVimeoPlayer(iframe);
+    observeOnHidden(iframe, () => stopVideo(iframe));
+
+}
+
+function loadVimeoPlayer(iframe: HTMLIFrameElement) {
+    const attachVimeoPlayer = () => {
+        const player = new window.Vimeo.Player(iframe);
+        player.on('play', function() {
+            stopInnerVideos(document.body, iframe);
+        });
+    };
+
+    if (window.Vimeo) {
```

Comment:
See comment above - let's always load it async

Reply:
Do you mean tag.async = true? Good idea, thanks.

Reply:
Shouldn't matter much at this point but will still help.
I incorrectly assumed initially that `window.Vimeo` object is defined in included `@vimeo/player` package